### ISSUE TITLE
Redo audio formats and dumps for songs

### ISSUE THREAD

**Issue by PikaDude** [animal-crossing-music-extension/ac-music-extension#54]
Some songs, such as the ACNL music is stored at a crazy 499kbps, despite the fact that the original soundtrack from the games files only exists at 299kbps. This is substantially larger than required and is also insane for any audio format. Songs need to be reduced to 320kbps maximum to reduce file size. Some songs for other games are also of lower quality than what's available and also contain audible artefacts of their lower quality.

## Progress

- [x] **Animal Crossing (Sunny)**:
- ✔71.0mb (320kb/s)
- ✔audible quality is good
- ❌songs have been slightly slowed down
- [x] **Animal Crossing (Snowing)**:
- ✔68.9mb (320kb/s)
- ✔audible quality is good
- ❌songs have been slightly slowed down
- [x] **Animal Crossing (Cheery)**:
- ✔68.0mb (320kb/s)
- ✔audible quality is good
- ❌songs have been slightly slowed down
- [ ] **Animal Crossing (Events)**
- [x] **City Folk/Wild World (Sunny)**:
- ✔56.2mb (208kb/s)
- ⚠some songs have soft static and some have audible buzzing, there don't appear to be any high quality rips without these issues. doesn't appear to be incredibly noticeable unless you have the song at a deafening volume, and it's only a handful of songs, so i think it should be fine
- [x] **City Folk/Wild World (Snowing)**:
- ✔59.0mb (208kb/s)
- ⚠some songs have soft static and some have audible buzzing, there don't appear to be any high quality rips without these issues. doesn't appear to be incredibly noticeable unless you have the song at a deafening volume, and it's only a handful of songs, so i think it should be fine
- [x] **City Folk/Wild World (Raining)**:
- ✔55.4mb (208kb/s)
- ⚠some songs have soft static and some have audible buzzing, there don't appear to be any high quality rips without these issues. doesn't appear to be incredibly noticeable unless you have the song at a deafening volume, and it's only a handful of songs, so i think it should be fine
- [ ] **City Folk/Wild World (Events)**
- [x] **New Leaf (Sunny)**:
- ✔71.4mb (300kb/s)
- ✔audible quality is good
- [x] **New Leaf (Raining)**:
- ✔62.9mb (300kb/s)
- ✔audible quality is good
- [x] **New Leaf (Snowing)**: 
- ✔70.1mb (300kb/s)
- ✔audible quality is now good
- [ ] **New Leaf (Events)**
- [x] **KK (Live)**
- ⚠291mb (192/320/kb/s) (CF/WW songs are 192kb/s as a higher version isn't available, NL songs are about 100kb/s since I can't locate any higher quality dumps.)
- ✔audible quality is good
- ✔contains all kk songs from ac, ww/cf and nl (91 songs, hence the large size)
- [x] **KK (Aircheck)**
- ⚠173mb (208/320kb/s) (CW/WW songs are 208kb/s as a higher version isn't available, NL songs are about 100kb/s since I can't locate any higher quality dumps.)
- ✔audible quality is good
- ✔contains all kk songs from ac, ww/cf and nl (91 songs, hence the large size)

**Comment by jordigarcl** [comment]
Is there any way we can help with this?

**Comment by PikaDude** [comment]
Most of it has been done, all that's left is to do the events and implement the function, which I'll do eventually.

**Comment by jordigarcl** [comment]
As event support can be tracked in #53, should we close this issue?

**Comment by PikaDude** [comment]
No, this is to do with audio dumps and quality. There's still some issues with static in other songs.

**Comment by PikaDude** [comment]
The Animal Crossing songs have been slightly slowed down. We need to find a high quality dump/rip that ISN'T slowed down and redo all of the loop times.

Top track is a low-quality rip downloaded off of YouTube, the bottom track is ours.
![audacity_2020-02-12_13-33-27](https://user-images.githubusercontent.com/18642415/74298165-1114c700-4d9d-11ea-8c7d-bb5514d3e662.png)
https://erisly.is-pretty.cool/2YVczut.mp4

**Comment by Beans-McGoo** [comment]
I'm not sure if this is related to this topic, but in my (rather short) experience, the New Horizons music, or at least the 9 pm song, stops abruptly before looping instead of looping smoothly. There are at least 2 seconds of silence before it loops again. It sounds like it may even have been that those 2 seconds are a part of the song that was cut out accidentally and not just a long pause that was inserted. Is that an issue with the audio files, or just something that my computer is doing?

**Comment by PikaDude** [comment]
There is nothing wrong with the extension or its audio files. These files are extracted directly from the game, along with the times that they loop. This is most definitely your computer's doing.

**Comment by PikaDude** [comment]
> I'm not sure if this is related to this topic, but in my (rather short) experience, the New Horizons music, or at least the 9 pm song, stops abruptly before looping instead of looping smoothly. There are at least 2 seconds of silence before it loops again. It sounds like it may even have been that those 2 seconds are a part of the song that was cut out accidentally and not just a long pause that was inserted. Is that an issue with the audio files, or just something that my computer is doing?

There is nothing wrong with the extension or its audio files. These files are extracted directly from the game, along with the times that they loop. This is most definitely your computer's doing.
